docs: minot text edits and typo fixes (#78)
rostools/r-cubed-intermediate: sessions/functionals.qmd

3.  Combine the results to present them together (e.g. into a data frame
    that you can use to make a plot or table).

So when you split data into multiple groups, you create a list (or a
*vector*) that you can then use (with the *map* functional) to apply a

> statistical technique to each group through *vectorization*.This

Reviewer: ```suggestion
statistical technique to each group through *vectorization*. This
```

Wrangling](https://r-cubed-intro.rostools.org/sessions/data-management.html#managing-and-working-with-data-in-r)
session of the beginner course). The common usage of these verbs is
through acting on and directly using the column names (e.g. without `"`

> quotes around the column name), e.g.,
> `saliva_df |> select(cortisol_norm)`. But many `{dplyr}` verbs can also

Reviewer: ```suggestion
quotes around the column name like with
`saliva_df |> select(cortisol_norm)`). But many `{dplyr}` verbs can also
```

But instead, there is the `across()` function that works like `map()`
and allows you to calculate the mean across which ever columns you want.
In many ways, `across()` is similar to `map()`, particularly in the

> arguments you give it and in the sense that it a functional. But they

Reviewer: ```suggestion
arguments you give it and in the sense that it is a functional. But they
```
